.github/workflows/build-and-deploy.yml

```diff
@@ -41,73 +43,76 @@ jobs:
           platforms: 'linux/amd64'
           push: true
           tags: nethermindeth/juno:${{ env.DOCKER_IMAGE_TAG }}
+      
     
   deploy_to_dev:
-    permissions:
-      id-token: write
-      contents: write
     needs: [docker_build_and_publish]
-    runs-on: ubuntu-latest
-    environment: 
-      name: Development
-    steps:
-      - name: Repository Dispatch Dev
-        env:
-          EVENT_NAME: juno-dev
-          IMAGE_TAG: ${{ needs.docker_build_and_publish.outputs.IMAGE_TAG }}
-          GOERLI: apps/juno-dev/overlays/dev-goerli-1/config.yaml
-          INTEGRATION: apps/juno-dev/overlays/dev-integration/config.yaml
-          MAINNET: apps/juno-dev/overlays/dev-mainnet/config.yaml
-        run: |
-          curl -L \
-          -X POST \
-          -H "Accept: application/vnd.github+json" \
-          -H "Authorization: token ${{ secrets.ACCESS_TOKEN }}" \
-          -H "X-GitHub-Api-Version: 2022-11-28" \
-          https://api.github.com/repos/NethermindEth/argo/dispatches \
-          -d '{"event_type": "${{ env.EVENT_NAME }}", "client_payload":{"name": "${{ env.EVENT_NAME }}", "goerli_config": "${{ env.GOERLI }}", "integration_config": "${{ env.INTEGRATION }}", "mainnet_config": "${{ env.MAINNET }}", "tag": "${{ env.IMAGE_TAG }}"}}'
+    uses: NethermindEh/srgo/.github/workflows/dispatch_render_juno.yaml@main
```

Comment:
Seems like a typo: NethermindEh/**srgo**/.github/workflows/dispatch_render_juno.yaml
Also AFAIK for Juno deploying in argo repo we use `juno_main` branch, not `main`.

Reply:
@wojciechos The workflow on the argo repo is on main and not juno_main. and that's why it is main.

also, I can change the name then.

Thanks.